src/liquid/components/ld-tooltip/ld-tooltip.tsx

```diff
@@ -0,0 +1,242 @@
+import Tether from 'tether'
+import { Component, Element, h, Host, Listen, Prop } from '@stencil/core'
+import { getClassNames } from '../../utils/getClassNames'
+import '../../components' // type definitions for type checks and intelliSense
+
+type Position =
+  | 'bottom center'
+  | 'bottom left'
+  | 'bottom right'
+  | 'left bottom'
+  | 'left middle'
+  | 'left top'
+  | 'right bottom'
+  | 'right middle'
+  | 'right top'
+  | 'top center'
+  | 'top left'
+  | 'top right'
+
+const mapPositionToAttachment = (position: Position) =>
+  ({
+    'bottom center': 'top center',
+    'bottom left': 'top left',
+    'bottom right': 'top right',
+    'left bottom': 'bottom right',
+    'left middle': 'middle right',
+    'left top': 'top right',
+    'right bottom': 'bottom left',
+    'right middle': 'middle left',
+    'right top': 'top left',
+    'top center': 'bottom center',
+    'top left': 'bottom left',
+    'top right': 'bottom right',
+  }[position])
+
+const mapPositionToTargetAttachment = (position: Position) =>
+  ({
+    'left bottom': 'bottom left',
+    'left middle': 'middle left',
+    'left top': 'top left',
+    'right bottom': 'bottom right',
+    'right middle': 'middle right',
+    'right top': 'top right',
+  }[position] ?? position)
+
+let tooltipCount = 0
+
+/**
+ * @virtualProp ref - reference to component
+ * @virtualProp {string | number} key - for tracking the node's identity when working with lists
+ */
+@Component({
+  tag: 'ld-tooltip',
+  styleUrl: 'ld-tooltip.css',
+  shadow: false,
+})
+export class LdTooltip {
+  @Element() element: HTMLElement
+
+  /** Show arrow */
+  @Prop() arrow = false
+
+  /** Delay in ms until tooltip hides (only when trigger type is 'hover') */
+  @Prop() hideDelay = 0
```

Comment:
This is a camel case prop, so we need to use the `applyPropAliases` utility in this class. Please search for `applyPropAliases` in https://github.com/emdgroup-liquid/liquid/blob/develop/CONTRIBUTING.md#conventions for an explanation.